node-module-system/calculate.ts

```diff
@@ -0,0 +1,25 @@
+import { add } from './add.js';
+import { subtract } from './subtract.js';
+import { multiply } from './multiply.js';
+import { divide } from './divide.js';
+
+if (process.argv.length >= 5) {
+  const operand1: string = process.argv[2];
+  const operator: string = process.argv[3];
+  const operand2: string = process.argv[4];
+
+  switch (operator.toLowerCase()) {
+    case 'add':
+      console.log(`result: ${add(Number(operand1), Number(operand2))}`);
+      break;
+    case 'subtract':
+      console.log(`result: ${subtract(Number(operand1), Number(operand2))}`);
+      break;
+    case 'multiply':
+      console.log(`result: ${multiply(Number(operand1), Number(operand2))}`);
+      break;
+    case 'divide':
+      console.log(`result: ${divide(Number(operand1), Number(operand2))}`);
+      break;
+  }
+}
```

Comment:
This is a good solution but still could be improved:
- You convert to Number multiple times.
- You have the same code that formats the result multiple times
- While you do some error checking, you don't tell the user that they messed up.
- You don't catch, for example, a mis-typed operator